cvat/apps/iam/rules/annotationguides.rego

```diff
@@ -0,0 +1,101 @@
+package annotationguides
+
+import data.utils
+import data.organizations
+
+# input: {
+#     "scope": <"view"|"update"|"delete"|"create"> or null,
+#     "auth": {
+#         "user": {
+#             "id": <num>,
+#             "privilege": <"admin"|"business"|"user"|"worker"> or null
+#         },
+#         "organization": {
+#             "id": <num>,
+#             "owner": {
+#                 "id": <num>
+#             },
+#             "user": {
+#                 "role": <"owner"|"maintainer"|"supervisor"|"worker"> or null
+#             }
+#         } or null,
+#     },
+#     "resource": {
+#         "id": <num>,
+#         "organization": { "id": <num> or null },
+#         "task": {
+#             "owner": { "id": <num> or null },
+#             "assignee": { "id": <num> or null },
+#         },
+#         "project": {
+#             "owner": { "id": <num> or null },
+#             "assignee": { "id": <num> or null },
+#         },
+#     }
+# }
+
+is_task_owner {
+    input.resource.owner.id == input.auth.user.id
+}
+
+is_task_assignee {
+    input.resource.assignee.id == input.auth.user.id
```

Comment:
It seems that `input.resource` will never have `owner` and `assignee` attributes. I guess you wanted to write `resource.task`

Reply:
Done